packages/agent/src/display.ts

```diff
@@ -434,22 +437,35 @@ function safeCalculateJwkThumbprint(jwk: JwkJson): string | undefined {
   }
 }
 export function getAttributesAndMetadataForMdocPayload(namespaces: MdocNameSpaces, mdocInstance: Mdoc) {
+  const topLevelMetadataFields = ['issue_date', 'expiry_date']
+
   const attributes: CredentialForDisplay['attributes'] = Object.fromEntries(
-    Object.values(namespaces).flatMap((v) =>
-      Object.entries(v).map(([key, value]) => [key, recursivelyMapAttribues(value)])
-    )
+    Object.values(namespaces).flatMap((v) => {
+      return Object.entries(v)
+        .filter(([key]) => !topLevelMetadataFields.includes(key))
+        .map(([key, value]) => [key, recursivelyMapAttributes(value)])
+    })
   )
 
+  // FIXME: Date should be fixed in Mdoc library
```

Comment:
@TimoGlastra fyi this is also currently an issue in the deployed app

Reply:
There seems to be more issues with dates, only in MDOC credentials. The expiry date in the nested driving privileges is also 'Invalid date', while it is correct in the sd-jwt version.